Source/Widgets/Timeline/Timeline.js

```diff
@@ -251,6 +253,11 @@ define([
         }
         //>>includeEnd('debug');
 
+        var timesValid = JulianDate.greaterThanOrEquals(startTime, Iso8601.MINIMUM_VALUE) && JulianDate.lessThanOrEquals(stopTime, Iso8601.MAXIMUM_VALUE);
```

Comment:
I'm not sure what you mean by plug-ins, by "our own code" I just meant `Viewer.js` (and maybe within the Timeline itself if it calls it).

We already crashed in this case, right?  So the only real change here is being more explicit about the error.